src-test/org/etools/j1939_84/controllers/part03/Part03Step03ControllerTest.java

```diff
@@ -81,17 +79,15 @@
     @Mock
     private VehicleInformationModule vehicleInformationModule;
 
-    private DataRepository dataRepository;
-
     private StepController instance;
 
     @Before
     public void setUp() throws Exception {
         DateTimeModule.setInstance(new TestDateTimeModule());
         DateTimeModule dateTimeModule = DateTimeModule.getInstance();
-        dataRepository = DataRepository.newInstance();
+        DataRepository dataRepository = DataRepository.newInstance();
```

Comment:
This is creating a new instance of the `dataRepository`, which isn't shared with the test.  How is this test passing?